README.md

```diff
@@ -195,371 +206,407 @@ This file contains sensitive information. Never share or commit this file to ver
+
+    -   Set to `true` to include remote jobs, `false` to exclude them
+
+-   `hybrid: [true/false]`
+
+    -   Set to `true` to include hybrid jobs, `false` to exclude them
+
+-   `onsite: [true/false]`
+
+    -   Set to `true` to include onsite jobs, `false` to exclude them
+
+-   `experience_level:`
+
+    -   Set desired experience levels to `true`, others to `false`
 
-  - Set to `true` to include remote jobs, `false` to exclude them
+-   `job_types:`
 
-- `hybrid: [true/false]`
+    -   Set desired job types to `true`, others to `false`
 
-  - Set to `true` to include hybrid jobs, `false` to exclude them
+-   `date:`
 
-- `onsite: [true/false]`
+    -   Choose one time range for job postings by setting it to `true`, others to `false`
 
-  - Set to `true` to include onsite jobs, `false` to exclude them
+-   `positions:`
 
-- `experience_level:`
+    -   List job titles you're interested in, one per line
+    -   Example:
 
-  - Set desired experience levels to `true`, others to `false`
+        ```yaml
+        positions:
+            - Software Developer
+            - Data Scientist
+        ```
 
-- `job_types:`
-  - Set desired job types to `true`, others to `false`
+-   `locations:`
 
-- `date:`
-  - Choose one time range for job postings by setting it to `true`, others to `false`
+    -   List locations you want to search in, one per line
+    -   Example:
 
-- `positions:`
-  - List job titles you're interested in, one per line
-  - Example:
+        ```yaml
+        locations:
+            - Italy
+            - London
+        ```
+
+-   `apply_once_at_company: [True/False]`
+
+    -   Set to `True` to apply only once per company, `False` to allow multiple applications per company
+
+-   `distance: [number]`
+
+    -   Set the radius for your job search in miles
+    -   Example: `distance: 50`
+
+-   `companyBlacklist:`
+
+    -   List companies you want to exclude from your search, one per line
+    -   Example:
+
+        ```yaml
+        companyBlacklist:
+            - Company X
+            - Company Y
+        ```
+
+-   `titleBlacklist:`
+
+    -   List keywords in job titles you want to avoid, one per line
+    -   Example:
+
+        ```yaml
+        titleBlacklist:
+            - Sales
+            - Marketing
+        ```
+
+#### 2.1 config.py - Customize LLM model endpoint
+
+-   `LLM_MODEL_TYPE`:
+    -   Choose the model type, supported: openai / ollama / claude / gemini / groq
+-   `LLM_MODEL`:
+    -   Choose the LLM model, currently supported:
+        -   openai: gpt-4o
+        -   ollama: llama2, mistral:v0.3
+        -   claude: any model
+        -   gemini: any model
+        -   azure: leave blank. your model deployment below define the model
+        -   groq: llama3-groq-70b-8192-tool-use-preview, llama3-groq-8b-8192-tool-use-preview, llama-3.1-70b-versatile, llama-3.1-8b-instant, llama-3.2-3b-preview, llama3-70b-8192, llama3-8b-8192, mixtral-8x7b-32768
+        -   aiml: any model
+-   `llm_api_url`:
```

Comment:
Can you change this to upper case to make it consistent with the variables in the config.py file? Do that for the other variables below as well.

Reply:
Done